### ISSUE TITLE
SmartCity-Accessibility Server Side #409 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#248]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [Ariel Kolikant] c12f4ef3f8eac38db9919a02cd94fe2ae975c70f - git is not allowing me to puul despite not having changes...this is

- [Ariel Kolikant] f21a766383848324c0c70e82e56b0c3393c4c46d - throws exception if user uses the name of the string splitter for some

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/409/)

**Comment by Kolikant** [comment]
I prematurely pressed push rather than commit, don't worry, easy fix

**Comment by JuanJenkinsBot** [comment]
Build was fixed!

**Comment by Kolikant** [comment]
You can sleep now sweet jenkins
The bad build can't hurt you anymore
